src/command_modules/azure-cli-appservice/setup.py

```diff
@@ -34,7 +34,7 @@
     'azure-cli-core',
     'azure-mgmt-web==0.32.0',
     # v1.17 breaks on wildcard cert https://github.com/shazow/urllib3/issues/981
-    'urllib3[secure]==1.16',
+    'urllib3[secure]>=1.18',
```

Comment:
Yes, but like I mentioned when Hao made the change last year, urllib3 had not released a fix yet, and `1.17` was the highest at that time.